### ISSUE TITLE
wrong implementation of the generated methods of delete requests when body is required 

### ISSUE THREAD

**Issue by ahmedelsayed-93** [Jumpscale/go-raml#280]
### raml code
```raml
delete:
            (DirectAccess):
            description: Delete file from container
            displayName: FileDelete
            body:
              application/json:
                type: DeleteFile
            responses:
              204:
                description: File deleted successfully
```

### generated python code
there generated method should expect data parameter in args 

```python
def FileDelete(self, containername, nodeid, headers=None, query_params=None, content_type="application/json"):
        """
        Delete file from container
        It is method for DELETE /nodes/{nodeid}/containers/{containername}/filesystem
        """
        uri = self.client.base_url + "/nodes/"+nodeid+"/containers/"+containername+"/filesystem"
        return self.client.delete(uri, headers, query_params, content_type)
```

**Comment by iwanbk** [comment]
Thanks for the report.
The generator actually always generate the code with no request body.

**Comment by iwanbk** [comment]
fixed by #283
